aggregate_logging_sizing.adoc

```diff
@@ -0,0 +1,283 @@
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+The https://github.com/openshift/openshift-docs/blob/master/install_config/aggregate_logging.adoc#overview[ElasticSearch, Fluentd and Kibana] (EFK) stack aggregates logs from nodes and applications running inside your openshift installation.
+Once deployed it uses http://www.fluentd.org/architecture[Fluentd] to aggregate event logs from all nodes, projects and pods into https://www.elastic.co/products/elasticsearch[Elasticsearch]. It also provides a centralized https://www.elastic.co/guide/en/kibana/current/introduction.html[Kibana] web UI where users and administrators can create rich visualizations and dashboards with the aggregated data.
+
+Fluentd https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-bulk.html[bulk uploads] logs to an index, in json format, and ElasticSearch then routes your search requests to the appropriate shards.
+
+== Installation
+The general procedure for installing an aggregate logging stack in Openshift Enterprise is described in https://docs.openshift.com/enterprise/3.2/install_config/aggregate_logging.html[Aggregating Container Logs].
+There are some important things to keep in mind while going through the installation guide:
+
+In order for the logging pods to spread evenly across your cluster, an empty https://docs.openshift.com/enterprise/3.2/admin_guide/managing_projects.html#using-node-selectors[node selector] should be used.
+
+----
+$ oadm new-project logging --node-selector=""
+----
+
+In conjunction with node labeling, which we will do later, this controls pod placement across the logging project. +
+You can now create the logging project.
+
+----
+$ oc project logging
+----
+
+If you are installing in a PoC or testing environment, a local openshift-ansible template install is recommended.
+
+----
+$ oc create -f ${OPENSHIFT_ANSIBLE_REPO}/roles/openshift_examples/files/examples/${VERSION}/infrastructure-templates/origin/logging-deployer.yaml
+----
+
+Elasticsearch (ES) should be deployed with a cluster size of at least three for resiliency to node failures (1 master/data node and 2 data nodes). This is specified by passing the *ES_CLUSTER_SIZE* parameter to the installer.
+
+----
+$ oc new-app logging-deployer-template \
+                        --param ES_CLUSTER_SIZE=3 \
+                        --param PUBLIC_MASTER_URL=$PUBLIC_MASTER_URL \
+                        --param MASTER_URL=$MASTER_URL \
+                        --param KIBANA_HOSTNAME=$KIBANA_URL
+----
+
+Refer to https://docs.openshift.com/enterprise/3.2/install_config/aggregate_logging.html#deploying-the-efk-stack[Deploying the EFK Stack] for a full list of parameters. +
+ +
+If you don't have an existing Kibana installation you can use *kibana.example.com* as a value to `KIBANA_HOSTNAME`. +
+Now ensure Fluentd pod spreading through labeling.
+
+----
+$ oc label nodes --all logging-infra-fluentd=true
+----
```

Comment:
Hmm, want to make sure that we're not largely duplicating the install documentation.